python/ql/src/Security/CWE-022/PathInjection.ql

```diff
@@ -1,7 +1,7 @@
 /**
  * @name Uncontrolled data used in path expression
  * @description Accessing paths influenced by users can allow an attacker to access unexpected resources.
- * @kind problem
+ * @kind path-problem
```

Comment:
We should probably have a change note about the conversion of some queries to path queries.